ribs/archives/_archive_base.py

```diff
@@ -60,12 +59,11 @@ class ArchiveBase(ABC):  # pylint: disable = too-many-instance-attributes
     This class assumes all archives use a fixed-size container with cells that
     hold (1) information about whether the cell is occupied (bool), (2) a
     solution (1D array), (3) objective function evaluation of the solution
-    (float), (4) behavior space coordinates of the solution (1D array), and (5)
+    (float), (4) measure space coordinates of the solution (1D array), and (5)
     any additional metadata associated with the solution (object). In this
     class, the container is implemented with separate numpy arrays that share
-    common dimensions. Using the ``cells`` and ``behavior_dim`` arguments in
-    ``__init__`` and the ``solution_dim`` argument in ``initialize``, these
-    arrays are as follows:
+    common dimensions. Using the ``solution_dim``, ``cells`, and
+    ``behavior_dim`` arguments in ``__init__``, these arrays are as follows:
```

Comment:
should we be using ``measure_dim`` instead of ``behavior_dim``?

Reply:
`measure_dim` for now until we change the `behavior_dim` property